### ISSUE TITLE
CodeView parsing in public crate interface

### ISSUE THREAD

**Issue by osiewicz** [willglynn/pdb#108]
Hi,
I work on toy Windows linker written in Rust. I would like to use this crate for creation of pdbs (I am aware that writing PDBs is not currently supported per #16). Before writing PDB I have to parse & dedup .debug$S and .debug$T records in object files, which contain CodeView-related information.

Would it make sense from your PoV to roll out CodeView parsing as a part of `pdb`'s public API (perhaps gated behind a feature flag)? I understand that's not the main use case of `pdb`, however IMHO it would make sense for `pdb` to host this kind of functionality.
I'd be happy to work on API changes and eventually #16 down the road.

**Comment by shuffle2** [comment]
Agree; I recently wanted to just parse these sections (no writing), and couldn't easily find something..
